src/Installer/redist-installer/targets/GenerateBundledVersions.targets

```diff
@@ -387,7 +387,11 @@
 
       <!-- The subset of ILCompiler target RIDs that are officially supported. Should be a subset of
            https://github.com/dotnet/runtime/blob/main/src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/ILCompilerRIDs.props -->
-      <ILCompilerSupportedRids Include="@(Net90ILCompilerSupportedRids)" />
+      <ILCompilerSupportedRids Include="
```

Comment:
All other lists have a version number. Should this one have a version number as well?

Reply:
this is fine. we need one item that represents the "current" list, this is that.

Reply:
Should other community supported architectures be moved to the "current" list?

Reply:
Next version properties added around preview 1 I think. Currently, net10 is using version less properties.

> Should other community supported architectures be moved to the "current" list?

It is the inheritance chain. For each component, the RID is listed once under the .NET version where the its support was enabled. The next version just inherits the previous. LoongArch was handled last year https://github.com/dotnet/installer/pull/19469/files.

Reply:
I think we add the next version properties when we do branching, i.e. for net11